server/src/core/util/date-time.ts

```diff
@@ -45,8 +45,8 @@ function makeDateTimeResolver<T>(
 
 export const dateTimeResolvers: Resolvers = {
     DateTime: {
-        utc: makeDateTimeResolver(t => t.toUTC().toISO()),
-        local: makeDateTimeResolver(t => t.toISO()),
+        utc: makeDateTimeResolver(t => t.toUTC().toISO() ?? `invalid date-time`),
```

Comment:
Would be better to use a `fail` function which always throws. It should be defined somewhere...

Reply:
It's here https://github.com/turingarena/turingarena/blob/ef4601a360bfaf24b2c7975ab4456d8357badd9e/server/src/main/api-context.ts#L67
but it needs an `ApiContext`.
One day we should refactor it to be just a free function (as it is in the client code).